Lombiq.NodeJs.Extensions/Lombiq.NodeJs.Extensions.targets

```diff
@@ -106,7 +106,8 @@
     <Copy SourceFiles="$(NodeJsExtensionsNpmPackageSourcePath)/config/consumer/.eslintrc.project.json"
           DestinationFiles="$(MSBuildProjectDirectory)/.eslintrc.json"
           Condition="
-            !Exists($(NodeJsExtensionsGlobalESLintConfigurationDirectory))
+            '$(NodeJsExtensionsGlobalESLintConfigurationDirectory)' != ''
```

Comment:
Unfortunately, this was a too quick shot - because with that change the default behavior, where you don't overwrite anything, will not work anymore.

I think it would be good for NE opt out of creating the file in case where the Scripts pipeline won't be executed due to missing input, i.e. non-existent folder or files in them.

But for the scope of this issue, please use `'false'` as the discriminating value instead of `''` because that at least won't break the default, project based scenario. Update the value in the OCC project file accordingly.

I apologize for the inconvenience.